crates/iota-proxy/src/middleware.rs

```diff
@@ -104,7 +103,6 @@ pub async fn expect_valid_public_key(
 #[derive(Debug)]
 pub struct LenDelimProtobuf(pub Vec<MetricFamily>);
 
-#[async_trait]
```

Comment:
Is this really okay?

Reply:
Yes, it is no longer needed, and not even exported by axum. I suppose their usage was basically solved when `async fn` got "recently" stabilized

Reply:
I didn't realize it was stabilized!